public/app/plugins/panel/geomap/style/types.ts

```diff
@@ -127,6 +127,22 @@ export interface StyleConfigValues {
   textConfig?: TextStyleConfig;
 }
 
+export enum GeoJSONPolyStyles {
```

Comment:
Nit: most other places in our codebase I think we capitalize `enum` properties.

Reply:
Good point. I wanted it to match the [StyleConfigValues properties](https://github.com/grafana/grafana/blob/18b9c8fd5fda882013e5224e48ce029d8259aee7/public/app/plugins/panel/geomap/style/types.ts#L116), but I suppose there may be a cleaner way to approach this.